daemon/algod/api/algod.oas2.json

```diff
@@ -549,6 +549,17 @@
             "in": "path",
             "required": true
           },
+          {
+            "type": "string",
+            "enum": [
+              "sha512_256",
+              "sha256"
```

Comment:
Would merkle-tree or vector-commitment be better names here? That seems like it would be more useful/descriptive than the generic hashing function being used to represent these things.

Reply:
Both are necessary to verify the proof, but the SHA512_256 not being supported natively on the EVM is the reason we've added this parameter, vector commitment is just an optimization of merkle tree that's a bit more secure cryptographically.

Reply:
So it's actually both, the different hash function was a necessity, while the different algorithm is used since it it's a "better" implementation of merkle tree (it's not actually necessary but since it has been integrated it should almost always be used instead of a standard merkle tree).
Do you think it should be more explicit in the parameter? 
```
"sha512_256_merkle",
"sha256_vector"
```